Source/ASTableView.mm

```diff
@@ -115,7 +115,9 @@ - (void)setElement:(ASCollectionElement *)element
   if (node) {
     self.backgroundColor = node.backgroundColor;
     self.selectedBackgroundView = node.selectedBackgroundView;
+#if TARGET_OS_IOS
     self.separatorInset = node.separatorInset;
+#endif
     self.selectionStyle = node.selectionStyle; 
     self.focusStyle = node.focusStyle;
```

Comment:
@alexhillc I had to do a manual merge of this file. Does this code look correct now / still?

Reply:
@appleguy, looks correct to me